apps/aega/src/aega_meta_tx.erl

```diff
@@ -92,7 +92,7 @@ gas(#ga_meta_tx{gas = Gas}) ->
 
 -spec gas_limit(tx(), non_neg_integer()) -> amount().
 gas_limit(#ga_meta_tx{gas = Gas, tx = InnerTx}, Height) ->
-    aetx:gas_limit(aetx_sign:tx(InnerTx), Height) + Gas.
+    aetx:inner_gas_limit(aetx_sign:tx(InnerTx), Height) + Gas.
```

Comment:
I see this change is for [not accounting twice size of inner tx for the sake of micro block gas limit](https://www.pivotaltracker.com/n/projects/2124891/stories/166018387). So strictly speaking this is done. Ok.